src/Three.WebGPU.Nodes.js

```diff
@@ -1,203 +1,2 @@
-export { DiscreteInterpolant } from './math/interpolants/DiscreteInterpolant.js';
-export { CubicInterpolant } from './math/interpolants/CubicInterpolant.js';
-export { Interpolant } from './math/Interpolant.js';
-export { Triangle } from './math/Triangle.js';
-export { MathUtils } from './math/MathUtils.js';
-export { Spherical } from './math/Spherical.js';
-export { Cylindrical } from './math/Cylindrical.js';
-export { Plane } from './math/Plane.js';
-export { Frustum } from './math/Frustum.js';
-export { Sphere } from './math/Sphere.js';
-export { Ray } from './math/Ray.js';
-export { Matrix4 } from './math/Matrix4.js';
-export { Matrix3 } from './math/Matrix3.js';
-export { Matrix2 } from './math/Matrix2.js';
-export { Box3 } from './math/Box3.js';
-export { Box2 } from './math/Box2.js';
-export { Line3 } from './math/Line3.js';
-export { Euler } from './math/Euler.js';
-export { Vector4 } from './math/Vector4.js';
-export { Vector3 } from './math/Vector3.js';
-export { Vector2 } from './math/Vector2.js';
-export { Quaternion } from './math/Quaternion.js';
-export { Color } from './math/Color.js';
-export { ColorManagement } from './math/ColorManagement.js';
-export { SphericalHarmonics3 } from './math/SphericalHarmonics3.js';
-export { SpotLightHelper } from './helpers/SpotLightHelper.js';
-export { SkeletonHelper } from './helpers/SkeletonHelper.js';
-export { PointLightHelper } from './helpers/PointLightHelper.js';
-export { HemisphereLightHelper } from './helpers/HemisphereLightHelper.js';
-export { GridHelper } from './helpers/GridHelper.js';
-export { PolarGridHelper } from './helpers/PolarGridHelper.js';
-export { DirectionalLightHelper } from './helpers/DirectionalLightHelper.js';
-export { CameraHelper } from './helpers/CameraHelper.js';
-export { BoxHelper } from './helpers/BoxHelper.js';
-export { Box3Helper } from './helpers/Box3Helper.js';
-export { PlaneHelper } from './helpers/PlaneHelper.js';
-export { ArrowHelper } from './helpers/ArrowHelper.js';
-export { AxesHelper } from './helpers/AxesHelper.js';
-export * from './extras/curves/Curves.js';
-export { Shape } from './extras/core/Shape.js';
-export { Path } from './extras/core/Path.js';
-export { ShapePath } from './extras/core/ShapePath.js';
-export { CurvePath } from './extras/core/CurvePath.js';
-export { Curve } from './extras/core/Curve.js';
-export { Controls } from './extras/Controls.js';
-export { DataUtils } from './extras/DataUtils.js';
-export { ImageUtils } from './extras/ImageUtils.js';
-export { ShapeUtils } from './extras/ShapeUtils.js';
-//export { PMREMGenerator } from './extras/PMREMGenerator.js';
-//export { WebGLUtils } from './renderers/webgl/WebGLUtils.js';
-export { createCanvasElement } from './utils.js';
-export * from './constants.js';
-export * from './Three.Legacy.js';
-
-export * from './materials/nodes/NodeMaterials.js';
-export { default as WebGPURenderer } from './renderers/webgpu/WebGPURenderer.Nodes.js';
```

Comment:
The WebGPU Nodes version has a `.Nodes.js` suffix in the file name, it is not present.

```js
export { default as WebGPURenderer } from './renderers/webgpu/WebGPURenderer.Nodes.js';
```

Reply:
Interesting, and it seems WebGPU Nodes doesn't export `BundleGroup`. I'm assuming that much is also a mistake, but good to know they aren't identical.

Reply:
Why do we need Renderer/Core-related classes like WebGPURenderer and StorageBufferAttribute in three/tsl? Wouldn't this cause confusion and redundancy with imports between `three/webgpu` and `three/tsl`?


In the package, aren’t these exports essentially placeholders?
```js
    "./webgpu": "./build/three.webgpu.js",
    "./tsl": "./build/three.webgpu.js"
```

Shouldn't it instead be:
```js
    "./webgpu": "./build/three.webgpu.js",
    "./tsl": "./build/three.webgpu.nodes.js"
```
where tsl only extends three.webgpu by adding node functionality, without duplicating the core WebGPU classes?

Reply:
The build `/three.webgpu.nodes.js` wouldn't exactly be a TLS package, it's an alternative to the standard `WebGPURenderer`, but without the shaders of the injected materials https://github.com/mrdoob/three.js/pull/28328. The project must include one or the other, never both.